.github/scripts/report-failed-tests.ps1

```
@@ -0,0 +1,155 @@
+  if ($counters) {
+    $total    = [int]$counters.total
+    $executed = [int]$counters.executed
+    if ($total -gt 0 -and $executed -lt $total) {
+      Add-Summary ''
+      Add-Summary (":warning: **Only $executed of $total discovered tests executed -- $($total - $executed) never ran.** " +
+                   'This shard is TRUNCATED; the failure list is a floor, not a total.')
+    }
+  }
+} catch {
+  # Reporter, never a gate -- a malformed TRX must not mask the failures we did parse.
+}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)